presto-orc/src/main/java/com/facebook/presto/orc/reader/BooleanBatchStreamReader.java

```diff
@@ -177,7 +173,6 @@ public String toString()
     @Override
     public void close()
```

Comment:
Do we still need this? (Same for all other `close`s)

Reply:
The close() methods in all XXXBatchStreamReader and BatchStreamReader should all be removed.